migrations/20240311202510-create-workspace-default-team-fk.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
         table: referencedTableName,
         field: 'id',
       },
+      onDelete: 'CASCADE',
```

Comment:
I noticed I was not able to remove workspaces from the database.  workspace_users and items are OK if you delete those before deleting workspaces rows. However, workspaces_teams and workspaces tables depends on each other so you are not able to delete neither.